src/Addons.php

```diff
@@ -265,4 +257,54 @@ protected static function get_action_button( $link, $status, $source, $slug ) {
 		</span>
 		<?php
 	}
+
+	/**
+	 * Print the content.
+	 *
+	 * @param array $item The item to print.
+	 * @param bool  $ignore_p_tag Whether to ignore the p tag.
+	 *
+	 * @return string
+	 */
+	public function print_content( $item, $ignore_p_tag = false ) {
+		$type = $item['type'];
+
+		$text = '';
+
+		switch ( $type ) {
+			case 'text':
+				$text = self::get_text( $item );
+				break;
+			case 'link':
+				$text = self::get_link( $item );
+				break;
+			case 'link_text':
+				$text = self::get_link_text( $item );
+				break;
+			case 'list':
+				$list_items = $item['items'];
+				$list       = '<ul class="krokedil_settings__list">';
+				foreach ( $list_items as $list_item ) {
+					$list .= '<li>';
+					$list .= $this->print_content( $list_item, true );
+					$list .= '</li>';
+				}
+				$list .= '</ul>';
+
+				$text = $list;
+				break;
+			case 'spacer':
+				$ignore_p_tag = true;
+				$text         = '<div class="krokedil_settings__spacer"></div>';
+				break;
+			default:
+				return '';
+		}
+
+		if ( $ignore_p_tag ) {
+			return $text;
+		}
+
+		return '<p>' . $text . '</p>';
+	}
```

Comment:
This seems to be a duplicate function to the one in the support class as well. We can just move it into the Subsection trait instead. And possibly make it a static function call as well.